### ISSUE TITLE
-init_from not working?

### ISSUE THREAD

**Issue by aramic** [jcjohnson/torch-rnn#126]
When using the -init_from flag to recommence training from a given checkpoint, it appears to start over at Epoch 1.00. Is this expected behavior or am I missing something? I am training on a large dataset which has been taking days and am concerned to make sure to not be starting over at the beginning.

`th train.lua -input_h5 data/wdwtest.h5 -input_json data/wdwtest.json -init_from cv/checkpoint_28700.t7 -gpu -1`

Response:

`Running in CPU mode`
`Initializing from 	cv/checkpoint_28700.t7`
`Epoch 1.00 / 50, i = 1 / 1836900, loss = 9.349901`
`Epoch 1.00 / 50, i = 2 / 1836900, loss = 7.825408`
`Epoch 1.00 / 50, i = 3 / 1836900, loss = 6.707690`

**Comment by aramic** [comment]
Solved by setting -reset_iterations to 0.
